scripts/zkai_seq32_value_compatible_boundary_frontier_gate.py

```
@@ -0,0 +1,673 @@
+    validate_payload(payload)
+    return payload
+
+
+def pretty_json(value: Any) -> str:
+    return json.dumps(value, indent=2, sort_keys=True, allow_nan=False) + "\n"
+
+
+def atomic_write_text(path: pathlib.Path, text: str) -> None:
+    path.parent.mkdir(parents=True, exist_ok=True)
+    if path.exists() and path.is_symlink():
+        raise Seq32ValueCompatibleBoundaryFrontierError(f"refusing to write symlink: {path}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)